src/test/java/TypewiseAlert/TypewiseAlertTest.java

```diff
@@ -9,7 +9,8 @@ public class TypewiseAlertTest
     @Test
     public void infersBreachAsPerLimits()
     {
-      assertTrue(TypewiseAlert.inferBreach(12, 20, 30) ==
-        TypewiseAlert.BreachType.TOO_LOW);
+        assertTrue(InferBreach.inferBreach(12, 20, 30) == BreachTypeEnum.TOO_LOW);
```

Comment:
eiCoverage needs to be improved with more test cases. Please add tests to achieve maximum coverage.